guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -350,7 +350,7 @@ To pass parameters to a block use the <a href="#argument"><code>&lt;argument&gt;
     <pre>&lt;referenceContainer name="container.name" display="false" /&gt;</pre>
 </ul>  
 
-### &lt;move&gt; {#fedg_layout_xml-instruc_ex_mv}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.